translations/German/general-notes.md

```diff
@@ -0,0 +1,114 @@
+say *"Dieser Fehler tritt häufiger auf."*
+
+* "Das ist die Seite, wo der Quelltext steht." - the "*wo*" is regional,
+say *"Das ist die Seite, auf der der Quelltext steht."* instead.
+
+## Form of address
+
+**Use "Du" instead of "Sie".**
+
+For the reasons provided in #Formal_or_Informal, stick to *Du* (informal)
+instead of *Sie* (formal) when addressing the reader and
+remember to capitalize *Du*.
+
+## Form of instruction
+
+**Prefer imperative over constructions with auxiliary verbs.**
+
+For instructions, try to use the imperative (e.g. *"Gehe auf die Seite"* -
+*"Go to the page"*) instead of constructions with auxiliary verbs
+(e.g. *"Du musst auf die Seite ... gehen"* - *"You have to go the page ..."*).
+This keeps the phrases short, less stiff and avoids unnecessary addressings
+of the reader.
+
+## Rules for captions
+
+**Use continuous titles with verbs in infinitive form**
+
+To be consistent with other online platforms, use continuous captions for buttons,
+item titles, etc. with verbs in infinitive form,
+e.g. *Manage streams* - *Kanäle verwalten* instead of *Verwalte Kanäle*.
+
+## Concatenation of words
+
+**Try to avoid it.**
+
+German is famous for its concatenations of nouns
+(e.g. *Heizölrückstoßdämpfung*, which means *fuel oil recoil attenuation*).
+For the sake of correct rendering and simplicity, you should try to avoid such
+concatenations whenever possible, since they can break the layout of the Zulip
+frontend. Try to stick to a maximum length of 20 characters and listen to your
+intuition.
+
+* A term like *Tastaturkürzel* for *Keyboard shortcuts* is fine - it is shorter than
+20 characters and commonly used in web applications.
+
+* A term like *Benachrichtigungsstichwörter* for *Alert words* should not be used,
+it sounds odd and is longer than 20 characters. You could use "*Stichwörter, die
+mich benachrichtigen*" instead.
+
+## Anglicisms
+
+**Use them if other web apps do so and a teenager could understand the term.**
+
+Unlike other languages, German happily adapts modern words from English.
+This becomes even more evident in internet applications,
+so you should not be afraid of using them if they provide an advantage over
+the German equivalent. Take the following two examples as a reference:
+
+* Translating *Stream*: Use the German word *Kanal*, since it is just as short
+and used in other web apps.
+
+* Translating *Bot*: Use *Bot*, as a completely accurate German
+equivalent **doesn't** exist (e.g. *Roboter*) and the term *Bot* is not
+unknown to German speakers.
+
+## Special characters
+
+**Use "ä, ö, ü" and "ß" consistently.**
+
+While *ä, ö, ü* and *ß* are more and more being replaced by *ae, oe, ue* and
+*ss* in chats, forums and even websites, German translations who take use of
+the umlauts and *ß* have a more trustworthy appearance. For capitalizations,
+you can replace the *ß* by *ss*.
+
+## False friends
+
+**Watch out!**
+
+False friends for the translation from German to English include
```

Comment:
A false friend is a word in another language that is spelled or sounds similar to a word in one's own language, yet has a different meaning .
Take for example the English word map: it is similar to the german word Mappe, which actually means folder. A map in German is a Landkarte